packages/wdio/__tests__/wdio.test.ts

```diff
@@ -92,19 +97,44 @@ describe('integration test axe with WebdriverIO', () => {
 });
 
 describe('integration test @sa11y/wdio with WebdriverIO', () => {
+    // TODO (test): add tests overriding the default rules
     // Note: "expect"s are in the helper method "checkAccessible"
     /* eslint-disable jest/expect-expect */
     it('should throw no error for html with no a11y issues', async () => {
         await browser.url(htmlFileWithNoA11yIssues);
         await checkAccessible(0);
     });
 
+    it('should throw no error for element with no a11y issues', async () => {
+        await browser.url(htmlFileWithNoA11yIssues);
+        await checkAccessible(0, browser, `#${shadowDomID}`);
+    });
+
     it('should throw error for html with a11y issues', async () => {
         await browser.url(htmlFileWithA11yIssues);
         await checkAccessible(numA11yIssues);
     });
+
+    it('should throw error for element with a11y issues', async () => {
+        await browser.url(htmlFileWithA11yIssues);
+        await checkAccessible(1, browser, `#${domWithA11yIssuesBodyID}`);
+    });
     /* eslint-enable jest/expect-expect */
 
+    it('should throw error for non-existent element', async () => {
+        await browser.url(htmlFileWithA11yIssues);
+        let err: Error = new Error();
+        try {
+            // using an existing elem ID using selector without the '#' prefix
+            await assertAccessible(browser, domWithA11yIssuesBodyID);
```

Comment:
@trevor-bliss one of the minor annoyances I noticed is like you can see here the `browser` object needs to be passed in as well whenever a CSS selector is passed. Since JS doesn't support calling functions with named params, thinking of receiving an object for params and using destructuring to get the values. That way defaults can be overridden without specifying all positional arguments.

Reply:
Ah ok yeah if the global `browser` object works here that makes sense. You probably want to add the `?` to the parameter on the function definition to tell typescript it's optional.